ci/azure-pipelines.yml

```diff
@@ -52,16 +46,15 @@ variables:
 pool:
   vmImage: "ubuntu-latest"
 
-#
 # The stages and jobs, potential for rationalization and optimization
 # Keeping it simple and explict whilst we gain experience
+#
 stages:
   - stage: Build_and_test
     jobs:
       - job: main
-        steps:
-          - script: |
-               env | sort            
+        steps:                
+          - template: templates/build-data.yml
```

Comment:
ahhh so that's how things can be simplified...

Reply:
You can even share templates between repos which might be interesting for some stuff ...in the future!